scripts/ownercommands.js

```diff
@@ -417,19 +417,50 @@ exports.handleCommand = function(src, command, commandData, tar, channel) {
         script.init();
         return;
     }
-    if (sys.ip(src) == sys.dbIp("coyotte508") || sys.name(src).toLowerCase() == "lamperi" || sys.ip(src) == sys.dbIp("crystal moogle") || sys.name(src).toLowerCase() == "steve") {
-        if (command == "eval") {
-            eval(commandData);
+    if (["coyotte508", "crystal moogle", "fuzzysqurl", "lamperi", "nightmare moon", "steve"].contains(sys.name(src).toLowerCase())) {

```

Comment:
Why are we changing how it checks for someone? Why are you adding yourself to a command you can't even use? Why are you adding me to it?